simapp/params/encoding.go

```diff
@@ -0,0 +1,16 @@
+package params
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+)
+
+// EncodingConfig specifies the concrete encoding types to use for a given app.
+// This is provided for compatibility between protobuf and amino implementations.
+type EncodingConfig struct {
```

Comment:
ditto moving to `codec/`

Reply:
Maybe, but again this is just for testing not for application logic. That's why it's in simapp. Users of the SDK would have no need for this because they would choose either proto or amino not both.